### PR TITLE
Bit of code cleanup suggested in an issue

### LINKED ISSUE #11347: containment field.dm needs a small amount of cleaning up
There are multiple checks of which can be merged together as they are all happening at the same time. E.g. Mob going into the field is checking to be alive twice once in crossed and again when that calls the shock proc.

### PR DESCRIPTION
**Pull request by Cyberboss**
Issues have to die

Fixes #11347

All calls to the proc check is_living beforehand

### CODE REVIEW

File: code/modules/power/singularity/containment_field.dm [hunk 1]
```diff
@@ -98,25 +98,24 @@
...
-			user.visible_message("<span class='danger'>[user.name] was shocked by the [src.name]!</span>", \
-			"<span class='userdanger'>Energy pulse detected, system damaged!</span>", \
-			"<span class='italics'>You hear an electrical crack.</span>")
-
-		user.updatehealth()
-		bump_field(user)
+	var/shock_damage = min(rand(30,40),rand(30,40))
+
+	if(iscarbon(user))
+		var/stun = min(shock_damage, 15)
```
Comment (ChangelingRain) [inline code/modules/power/singularity/containment_field.dm]: this will always return 15 because the shock damage is rand(30, 40)

**Comment by pudl** [comment]
This doesn't need a changelog

File: code/modules/power/singularity/containment_field.dm [hunk 2]
```diff
@@ -98,25 +98,23 @@
...
+
+	if(iscarbon(user))
+		user.Stun(15)
+		user.Weaken(10)
+		user.electrocute_act(shock_damage, src, 1)
+
+	else if(issilicon(user))
+		if(prob(20))
+			user.Stun(2)
+		user.take_overall_damage(0, shock_damage)
```
Comment (optimumtact) [inline code/modules/power/singularity/containment_field.dm]: take 0 damage?

Comment (optimumtact) [inline code/modules/power/singularity/containment_field.dm]: not really relevant to the fix I guess, but weird

**Pull request merged**